### PR TITLE
Events display

### PR DESCRIPTION
**Pull request by chanteriam**
## Describe your changes
Makes the past + upcoming event data accurate to the user's events and allows users to review past events.

## Checklist before merging

- [x] My code produces no errors when run
- [x] My code is linted and properly formatted

### CODE REVIEW

**Review by sarahwalker10 — CHANGES_REQUESTED** [review]
1. I recommend changing the "Did you attend this event?" to a slide "Yes" + "No" option, bc right now it is not intuitive that I need to click "No" in oder to say "yes". 
Screenshot 2024-05-13 at 10 04 11 AM

2. I notice that when I receive a pop-up about an event I've been invited to, after i click "Sign Me Up!", the event does not appear in my "Upcoming events" until after I refresh the page.

File: backend/shoulder/s2s/db_models/user_events.py
Comment (ehabich) [inline backend/shoulder/s2s/db_models/user_events.py:1]: Please add 'rsvp' and 'attended' column descriptions into the docstring unless the backend team does this in a different PR

**Review by ehabich — COMMENTED** [review]
I just added one thing to be fixed in the documentation of the user_events.py model, but otherwise this looks great.

Nice to haves (but not necessary for this merge): 
1. removing star review highlight from Past Events that have already been reviewed
2. adding day of week to upcoming event dates

File: backend/shoulder/s2s/db_models/user_events.py
Comment (chanteriam) [reply backend/shoulder/s2s/db_models/user_events.py:1]: Implemented the changes. Added a screen recording to the frontend slack channel.

**Review by ehabich — APPROVED** [review]
Looks incredible!

**Pull request merged**